src/extras-openmoji/healthcare/E314.svg

```diff
@@ -0,0 +1,15 @@
+<svg id="emoji" viewBox="0 0 72 72" xmlns="http://www.w3.org/2000/svg">
```

Comment:
This emoji is currently listed as "appendix" but this is a large intestine emoji, can I change its name? Also I don't think the appendix translates well to an emoji as it's not very distinctive.